### DISCUSSION TITLE
Unable to Call Stored Procedure from Postman

### DISCUSSION THREAD

**Discussion by girishvenkata14** [PostgREST/postgrest#1742] [Q&A] [answered]
### Environment

* PostgreSQL version: "PostgreSQL 12.5 (Ubuntu 12.5-1.pgdg20.04+1) on x86_64-pc-linux-gnu, compiled by gcc (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0, 64-bit"
* PostgREST version: 
* Operating system:

### Description of issue
Unable to call the stored procedure from postman and getting below errors. Could you please help

Below is the SP used:
<img width="522" alt="SP" src="https://user-images.githubusercontent.com/77968207/105703156-d581f700-5f32-11eb-8c40-cbb6a56aeaac.PNG">

Below is the issue when calling stored procedure using Get in postman:
<img width="596" alt="Issue" src="https://user-images.githubusercontent.com/77968207/105703245-f21e2f00-5f32-11eb-92b6-ca4a451ef783.PNG">

Below is the issue when calling stored procedure using Post in postman
1) Using CALL
<img width="649" alt="Issue1" src="https://user-images.githubusercontent.com/77968207/105703308-08c48600-5f33-11eb-902e-78b456c5b5ab.PNG">
 
2)Using JSON
<img width="656" alt="Issue2" src="https://user-images.githubusercontent.com/77968207/105703327-0feb9400-5f33-11eb-8e12-b2d2899a33cf.PNG">

**Comment by wolfgangwalther** [comment]
Stored Procedures are not supported. You need to create this as a function, not a procedure.

**Comment by steve-chavez** [comment, marked as answer]
For reference: https://github.com/PostgREST/postgrest/issues/1210.

Postgres 11 procedures have limitations, AFAICT you don't need them unless you want to start a transaction inside them. So changing your proc to a function should work fine - `CREATE FUNCTION` instead of `CREATE PROCEDURE`.
